src/site/xdoc/server/quick-start-cassandra.xml

```diff
@@ -183,6 +185,124 @@ Step 9: Monitor
 
 </source>
 
+  <p>Second method, with docker-compose:</p>
+  
+<source>
+Step 0: Requirements
+####################
+
+  * Java 8 SDK
+  * 2GB RAM
+  * Docker 1.7.1+
+  * Maven 3.3
+
+Step 1: Download
+#################
+
+  * Get the James docker-compose yaml file
```

Comment:
I would just say:

"Get the James docker-compose file"

without "yaml" because it does not matter and moreover I think it's a "pleonasm"